test/index.test.jsx

```diff
@@ -421,4 +421,20 @@ describe('web components', () => {
 			'<h1>Light DOM Children</h1><div><slot><p>Child 1</p><p>Child 2</p></slot></div>'
 		);
 	});
+
+	it('supports the `serializable` option', async () => {
+		function SerializableComponent() {
+			return <div className="serializable-child">Serializable Shadow DOM</div>;
+		}
+
+		registerElement(SerializableComponent, 'x-serializable', [], {
+			shadow: true,
+			serializable: true,
+		});
```

Comment:
Mind registering this with `serializable` undefined (`x-unserializable` or something), checking that `.serializable` is false, and then checking this `true` case? I realize we don't mirror that with our other options but we should. Helps ensure things don't get accidentally enabled by default over time.

Both cases can be in the same test, just one after another.